jdaviz/configs/imviz/tests/test_linking.py

```diff
@@ -162,20 +159,25 @@ def test_wcslink_affine_with_extras(self):
 
         # Also check the coordinates display: Last loaded is on top.
 
-        self.viewer.on_mouse_or_key_event({'event': 'mousemove', 'domain': {'x': 0, 'y': 0}})
-        assert self.viewer.label_mouseover.pixel == 'x=01.0 y=-0.0'
-        assert self.viewer.label_mouseover.value == '+1.00000e+00 '
-        assert self.viewer.label_mouseover.world_ra_deg == '337.5202808000'
-        assert self.viewer.label_mouseover.world_dec_deg == '-20.8333330600'
+        label_mouseover = self.imviz.app.session.application._tools['g-coords-info']
+        label_mouseover._viewer_mouse_event(self.viewer,
+                                            {'event': 'mousemove',
+                                             'domain': {'x': 0, 'y': 0}})
+
+#        assert label_mouseover.as_text() == ('Pixel x=00.0 y=00.0 Value +1.00000e+00',
+#                                             'World 22h30m04.8674s -20d49m59.9990s (ICRS)',
+#                                             '337.5202808000 -20.8333330600 (deg)')
+#        for some reason getting:

```

Comment:
Isn't this correct? These are the values originally tested for. As the comment said above, last loaded is on top. What do you think is wrong here?

Reply:
ah, yes, sorry about that.  This was the same `00.0` vs `-0.0` problem from the other case and I just forgot to update and uncomment this one.  (The `x=01.0` is the same as the old test and reproduced here as well).